From 9e5f6bc4bb60f5d2a46c1955efb5b8b0c46d4a53 Mon Sep 17 00:00:00 2001
From: Olivier Brunel <jjk@jjacky.com>
Date: Tue, 15 Oct 2013 16:43:28 +0200
Subject: [PATCH 1/3] treeview: is_rubber_banding_active() only based on status

Whether a rubber banding operation is being done or not only comes from the
status, looking at whether it's enabled or not isn't needed.

This will also allow to have it disbaled, yet have a rubber banding operation
being done, when triggered manually.

Signed-off-by: Olivier Brunel <jjk@jjacky.com>
---
 gtk/gtktreeview.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/gtk/gtktreeview.c b/gtk/gtktreeview.c
index d228360..4389e16 100644
--- a/gtk/gtktreeview.c
+++ b/gtk/gtktreeview.c
@@ -15737,8 +15737,7 @@ gtk_tree_view_is_rubber_banding_active (GtkTreeView *tree_view)
 {
   g_return_val_if_fail (GTK_IS_TREE_VIEW (tree_view), FALSE);
 
-  if (tree_view->priv->rubber_banding_enable
-      && tree_view->priv->rubber_band_status == RUBBER_BAND_ACTIVE)
+  if (tree_view->priv->rubber_band_status == RUBBER_BAND_ACTIVE)
     return TRUE;
 
   return FALSE;
-- 
1.8.4

